nixos/modules/virtualisation/podman/default.nix

```diff
@@ -141,6 +142,10 @@ in
         };
       };
 
+      security.apparmor.includes."tunables/global" = ''
```

Comment:
Could you add a comment describing what this is trying to accomplish?
I'm not familiar enough with apparmor to understand this.

Reply:
Also, the same feedback as in my other comment applies, isn't this something that the apparmor module should add? (I don't know, this is a question, not a request for changes).

Reply:
Other things don’t really need it because the include in Podman’s policy itself works fine (as AA uses the proper load path), it’s just that Podman (and most other container tools, as they all roughly share this same code) *also* does it’s own lookup based on a hard-coded path. An alternative would be to patch Podman to do that differently, either in Nixpkgs or, ideally, upstream, but that has it’s own hurdles

The relevant bit in Podman is in [this code](https://github.com/containers/podman/blob/main/vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go#L52-L60).

I’m definitely open to whatever solution is thought best here.